src/components/FooterWrapper.js

```diff
@@ -40,8 +40,7 @@ const FooterWrapper = ({ children }) => {
       ]),
       div({ style: { flexGrow: 1 } }),
       div({ style: { fontWeight: 600, fontSize: '10px' } }, [
-        `Copyright ©${buildTimestamp.getFullYear()}. Built on `,
-        buildTimestamp.toLocaleString()
+        `Copyright ©${buildTimestamp.getFullYear()}.`
```

Comment:
Nit: looks a little strange to have a `.` after the Copyright year, maybe remove it?